geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/ingest/HBaseBulkIngestCommand.scala

```diff
@@ -55,9 +55,6 @@ class HBaseBulkIngestCommand extends HBaseIngestCommand {
         throw new NotImplementedError("Bulk ingest not implemented for local mode")
     }
   }
-
-  override protected def createShpIngest(): Runnable =
-    throw new NotImplementedError("Bulk ShpFile ingest not implemented")
```

Comment:
With this PR, what happens when people try to ingest shapefiles with MR?

Reply:
it will use local mode regardless of what is specified - if you use the bulk ingest command that will make it throw an IllegalArgumentException I believe. We haven't ever supported MR shapefile ingest, this doesn't change that